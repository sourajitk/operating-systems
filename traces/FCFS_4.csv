1,0,1
2,0,1
3,0,1
4,10,3
5,10,3
6,10,3
7,10,1
8,20,2
9,20,2
10,20,2
