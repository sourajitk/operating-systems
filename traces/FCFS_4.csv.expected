1,1
2,2
3,3
4,13
5,16
6,19
7,20
8,22
9,24
10,26
